test/utils/version/version.go

```diff
@@ -0,0 +1,29 @@
+package version
+
+import (
+	"context"
+	"fmt"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/operator-framework/api/pkg/lib/version"
+	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+)
+
+var defaultNamespace = "numaresources-operator"
+
+func OfOperator(ctx context.Context, cli client.Client) (*version.OperatorVersion, error) {
```

Comment:
What `OfOperator` stands for?
I had rather not to use non-standardize acronym/abbreviations in function names.

Reply:
I think is meant to read `version.OfOperator` which I generally like

Reply:
@ffromani @Tal-or yes I have aimed for this to be read this way `version.OfOperator`